test/18.js

```diff
@@ -1,34 +1,43 @@
 const expect = require('chai').expect;
 let solution1 = require('../solutions/18').solution1;
-    solution2 = require('../solutions/18').solution2;
-// solution = require('../yourSolution').solution;
+solution2 = require('../solutions/18').solution2;
+solution3 = require('../solutions/18').solution3;
 
 describe('Find missing number in array', () => {
-  it('should return 7 for [5,6,8,9]', () => {
-    const result1 = solution1([5, 6, 8, 9]);
-          result2 = solution2([5, 6, 8, 9]);
-    expect(result1).to.equal(7);
-    expect(result2).to.equal(7);
-  });
+it.only('should return 7 for [5,6,8,9]', () => {
```

Comment:
Please remove `only` from all of the test cases. and undo all the changes you introduced to spaces in the beginning of each line.